event_registration_cancel_reason/tests/test_event_registration_cancel_reason.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2016 Antiun Ingeniería S.L.
+# Copyright 2016 Tecnativa - Pedro M. Baeza
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import exceptions, fields
+from odoo.tests import common
+
+
+class TestEventRegistrationCancelReason(common.TransactionCase):
```

Comment:
You can improve speed using SavepointCase

Reply:
Done